config.py

```diff
@@ -120,6 +120,9 @@
 
 APP_HOST            = config_parser['APP']['APP_HOST']
 REQUIRE_MULTIPLE_APPROVALS = config_parser['APP'].getboolean('REQUIRE_MULTIPLE_APPROVALS', False)
+ALLOWED_APPROVERS = config_parser['APP'].get('allowed_approvers', None)
+if ALLOWED_APPROVERS:
+    ALLOWED_APPROVERS = ALLOWED_APPROVERS=ALLOWED_APPROVERS.split(',')
```

Comment:
Can we also add the config to the template https://github.com/teamsempo/SempoBlockchain/blob/master/config_files/public/local_config.ini